snippets/non-production-disclaimer.mdx

```diff
@@ -0,0 +1,3 @@
+<Warning>
+    Use this guide for testing purposes only and not for production deployments. The OpenOps team will be working on production deployment guides in the future, but for now, this guide is not intended for production use.
```

Comment:
Can we say instead something like, 

This guide is for testing and evaluation purposes only and not for production deployments. Please reach out to us at support@openops.com if you'd like information on how to set it up in a production environment.

Something like this?